### DISCUSSION TITLE
Access to file denied: insufficient permissions

### DISCUSSION THREAD

**Discussion by dvahn** [plausible/analytics#1639] [Self-Hosted Support]
I tried to run plausible on localhost to test it. The setup runs without any errors, but the plausible_events_db container stays on the status "Restarting". In the logs I get the following error:

_"Poco::Exception. Code: 1000, e.code() = 0, e.displayText() = Exception: Failed to merge config with '/etc/clickhouse-server/config.d/logging.xml': Access to file denied: insufficient permissions: /etc/clickhouse-server/config.d/logging.xml (version 21.3.2.5 (official build))"_

I tried rebuilding with "sudo docker compose up", still no luck.

Has anyone had this error before? 

Edit: 
By removing the following lines

```
- ./clickhouse/clickhouse-config.xml:/etc/clickhouse-server/config.d/logging.xml:ro
- ./clickhouse/clickhouse-user-config.xml:/etc/clickhouse-server/users.d/logging.xml:ro
```

in the docker-compose.yml I managed to fix the error and got plausible up and running. Still no idea why there are missing permissions inside the container...

**Comment by ghost** [comment]
I was also getting this, using podman-compose instead of docker-compose, but the fix you mentioned worked a treat. Will running Plausible without those lines create any issues down the line?
